### DISCUSSION TITLE
Whiskey, Steam ad SteamVR

### DISCUSSION THREAD

**Discussion by twickl** [Whisky-App/Whisky#770] [Q&A]
Dear Community,

with the Release of SteamVR Link in the Quest Store (removes the prerequisite to run Meta Quest Desktop Software) I wnated to check if SteamVR on Mac via Whiskey would be possible. So I installed the Windows Version of Steam in a Wine Bottle Windows 10 and after that SteamVR via Steam.
After changing the Wine DLL attributes of msvcp140 and vcruntime140 to "Native, Builtin" in the Library Tab of the Wine Configuration UI the Steam link app on the Quest 3 will try to connect to Steam running in Whiskey but will report that the Host-Computer is locked. Not the Mac is locked but the "Windows" seems to report as locked. 

Do you have any Idea how I can try to "unlock" the Host Computer?

**Comment by buftio** [comment]
I have the same issue!
@twickl have you found any solution?

**Comment by Wanneshey** [comment]
Same issue here. Has someone found a solution yet? I'm so close to running VR (I hope)

**Comment by marxo126** [comment]
same issue. Anyone found solution?

**Comment by scoobisofficial** [comment]
I cant even get steamvr to launch.

**Comment by DJPotatoBomb** [comment]
I have the same issue

**Comment by scoobisofficial** [comment]
ok, so i used VMWare Fusion to launch SteamVR, note, you do need ALVR.

**Reply by lukecarbis** [reply]
That's not really helpful for this particular issue in the Whiskey-App repo.

**Reply by TheBeginnerPro** [reply]
Hi friends, who had the error steamwebhelper not responding? How to solve it?

**Comment by TheBeginnerPro** [comment]
Hi friends, who had the error steamwebhelper not responding? How to solve it?

**Comment by scoobisofficial** [comment]
Doing some poking and prodding with AnyDesk, I found that there is no desktop to be seen or anything to be exact. I might try ALVR to see if there can even be a stream at all.
